dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -170,10 +170,13 @@ error.unique.value.conflict=CWWKG0031E: The value {1} specified for unique attri
 error.unique.value.conflict.explanation=The value for the specified attribute must be unique but it is already being used.
 error.unique.value.conflict.useraction=Change the attribute value so that it is unique.
 
-warn.config.invalid.value=CWWKG0032W: Unexpected value specified for property [{0}], value = [{1}]. Expected value(s) are: {2}.
+# {3} is either the formatted value of default.value.in.use or an empty string
```

Comment:
is {3} A property? a variable? Try  to specify... like  
**change**
{3} is either the formatted value of default.value.in.use or an empty string
**to**
The {3} property is either the formatted value of default.value.in.use or an empty string